conftest.py

```diff
@@ -1,12 +1,28 @@
 from player.models import BallGame, Player
+from court.models import Court
+from court_ball_game.models import CourtBallGame
+from decimal import Decimal
 import pytest
 
 
 @pytest.fixture
 def player():
     player = Player.create(
-        username='daniel',
+        username='Playerr',
```

Comment:
`Playerr` intended?

Reply:
Yes, that's what I meant, I'll change it to player.